alert.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="alert.css">
+</head>
+<script src="https://kit.fontawesome.com/a076d05399.js"></script>
+<body>
+    <header>
+        <h1 class="heading">BOOSTWEB UI</h1>
+        <div class="social-media-link">
+            <a href="#"> <i class="fab fa-instagram" style="font-size:40px"></i> </a>
+            <a href="#"> <i class="fab fa-linkedin" style="font-size:40px"></i></a>
+            <a href="#"> <i class="fab fa-github" style="font-size:40px"> </i></a>
```

Comment:
try to minimize repetitive codes. Use classes instead of inline styling wherever possible.